WordPress/Classes/Models/SearchResultsStatsRecordValue+CoreDataClass.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import CoreData
+
+
+public class SearchResultsStatsRecordValue: StatsRecordValue {
+
+    /// This is a magic value indicating that this item represents search result counts for
+    /// unknown/encrypted search terms.
+    public static let unknownSearchTermString = "WPiOS.search_string_unknown"
```

Comment:
It seems like this could be `internal`. Agreed?

Reply:
Yup! Fixed.